src/d3d12/d3d12_command_list.cpp

```diff
@@ -160,6 +160,15 @@ namespace wr::d3d12
 		cmd_list->m_native->SetDescriptorHeaps(num, n_heaps.data());
 	}
 
+	void BindComputePipeline(CommandList* cmd_list, PipelineState * pipeline_state)
+	{
+		if (pipeline_state->m_type == PipelineType::COMPUTE_PIPELINE) 
```

Comment:
Atleast warn if this happens (`LOGW`). Or just let it crash. The debug layer will warn what went wrong.

Reply:
Now logging a warning instead of doing nothing